docs/topics/auth/customizing.txt

```diff
@@ -82,8 +82,8 @@ backends that follow.
     authenticated user is needed. This effectively means that authentication
     sources are cached on a per-session basis, so if you change
     :setting:`AUTHENTICATION_BACKENDS`, you'll need to clear out session data if
-    you need to force users to re-authenticate using different methods. A simple
-    way to do that is simply to execute ``Session.objects.all().delete()``.
+    you need to force users to re-authenticate using different methods. A way
+    to do that is to execute ``Session.objects.all().delete()``.
```

Comment:
I think the sentence can start with "A simple way" still - there are certainly more complicated methods. The "simply" was definitely redundant though.